src/main/java/com/joy/joyfillandroid/DropDown.kt

```diff
@@ -0,0 +1,220 @@
+package com.joy.joyfillandroid
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.graphics.drawable.GradientDrawable
+import android.os.Build
+import android.text.InputType
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.KeyEvent.ACTION_UP
+import android.view.View
+import android.widget.CheckBox
+import android.widget.LinearLayout
+import androidx.appcompat.widget.AppCompatAutoCompleteTextView
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+
+class DropDown @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+
+    val autoCompleteTextView = AppCompatAutoCompleteTextView(context)
+    private lateinit var bottomSheetDialog: BottomSheetDialog
+
+    init {
+
+        addView(createAutoCompleteTextView(context))
+
+    }
+    fun createAutoCompleteTextView(context: Context): LinearLayout {
+        val linearLayout = LinearLayout(context)
+
+        linearLayout.orientation = LinearLayout.VERTICAL
+        linearLayout.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)
+        linearLayout.gravity = Gravity.START
+
+        val textField = TextField(context)
+        textField.text ="Dropdown Field"
```

Comment:
Same as other PR. The function autoCompleteTextView it seems odd that there is a direct text = "" type assignment here where each field is dynamic.